### PR TITLE
Fix app shell isolated

### PR DESCRIPTION
**Pull request by arthurcgc**
This PR makes the app shell isolated container declare it's resource requirements beforehand.
The resource requirements are based from the app plan.

### CODE REVIEW

**Comment by wpjunior** [comment]
LGTM @arthurcgc 🚀

File: provision/kubernetes/provisioner.go [hunk 1]
```diff
@@ -1445,6 +1447,17 @@ func runIsolatedCmdPod(ctx context.Context, client *ClusterClient, opts execOpts
...
+	var requirements *apiv1.ResourceList
+	memory := opts.app.GetMemory()
+	cpu := opts.app.GetMilliCPU()
+	if memory > 0 && cpu > 0 {
+		requirements = &apiv1.ResourceList{
+			apiv1.ResourceMemory: *resource.NewQuantity(memory, resource.BinarySI),
+			apiv1.ResourceCPU:    *resource.NewMilliQuantity(int64(cpu), resource.DecimalSI),
+		}
+	}
+
```
Comment (nettoclaudio) [inline provision/kubernetes/provisioner.go:1460]: I'd replace this code block in favor of the snippet below. Actually, you could extract it to a helper function and reuse the same logic to create the resource list in both places, WDYT?

https://github.com/tsuru/tsuru/blob/80db3599179e4fdc7e0d9876756342dce4f86e93/provision/kubernetes/deploy.go#L654-L677

Comment (nettoclaudio) [reply provision/kubernetes/provisioner.go:1460]: You could move this whole implementation to `helpers.go` file though.

**Review by nettoclaudio — CHANGES_REQUESTED** [review]
We can reuse the code to create the resource list from deploy stages, PTAL.

**Pull request merged**